### DISCUSSION TITLE
Using Overlay with Kick, Twitch, and YouTube at the same time?

### DISCUSSION THREAD

**Discussion by chrisfoose** [baffler/Transparent-Twitch-Chat-Overlay#314] [Q&A]
Is there a way to connect the program to Kick, Twitch, and YouTube all at the same time?

**Comment by yodanilo** [comment]
same here. Twitch chat seems to work the transparency but custom url no, like kick, gets a black chat window

**Reply by jwizfoshizz** [reply]
i want to make ticoding into for tiktok pop out how do i do that and make it transparent?

**Comment by User5678-crypto** [comment]
> Is there a way to connect the program to Kick, Twitch, and YouTube all at the same time?

Works if you create a widget with link to any chat aggregator like buttrix etc
